### DISCUSSION TITLE
F-Driod release

### DISCUSSION THREAD

**Discussion by tort32** [Vinetos/Tranquille#3] [Q&A]
I was using YACB for a few years and love to see your release at F-Droid.
Cheers 🥂

**Comment by Vinetos** [comment]
Hey ! 
I think I will be ready to publish the app on F-Droid at the end of the week :D !

**Comment by Vinetos** [comment]
I have created the request submission for submission of the app this day. I hope it will be accepted anytime soon :D !

**Reply by Cris5791** [reply]
Hi, I see that everything has been at a standstill for a few months, is there any news? Thanks

**Reply by Vinetos** [reply]
Hello, Yup this is mainly waiting for https://github.com/Vinetos/Tranquille/issues/6
I think by the end of the week the issue will be closed and the app ready for review on F-droid
